source/common/modules/persistent-data-container/index.ts

```diff
@@ -94,7 +94,7 @@ export default class PersistentDataContainer {
   public async isInitialized (): Promise<boolean> {
     try {
       await fs.access(this._filePath, FSConstants.R_OK | FSConstants.W_OK)
-      return true
+      return (await this.get()) !== null
```

Comment:
Sorry for my late response! I would advise against calling `get` here in this function to keep it clean. Rather, I would look at the actual file contents and see if it's an empty file (using `fs.readfile`). Anything else (invalid syntax) will throw an error anyways.